pkg/plugins/runtime/k8s/containers/factory.go

```diff
@@ -108,6 +108,17 @@ func (i *DataplaneProxyFactory) NewContainer(
 			"--concurrency="+strconv.FormatInt(cpuCount, 10))
 	}
 
+	lifecycle := kube_core.Lifecycle{}
+	if i.ContainerConfig.LifeCycle.SidecarInitTimeout > 0 {
+		lifecycle = kube_core.Lifecycle{
+			PostStart: &kube_core.LifecycleHandler{
+				Exec: &kube_core.ExecAction{
+					Command: []string{"command goes here"},
```

Comment:
@kumahq/kuma-maintainers The question is where should we add a command that is going to check if the sidecar is ready? There is a few options:
* `kumactl` - it feels like the right place for checking health but we don't include `kumactl` binary in `kuma-dp` image
* `kuma-dp wait` - we could include a script that checks if sidecar is ready in `kuma-dp` 
* script that runs on the container(bash) - might be problematic because we are using light images without most of the additional packages

What are your opinions?

Reply:
I also prefer kuma-dp wait